index.html

```diff
@@ -0,0 +1,273 @@
+                        </div>
+                        <h3>Skills</h3>
+                        <ul class="about-categories">
+                            <li>Codekit</li>
+                            <li>GitHub</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>      
+        </section>
+         <!-- Contact Section -->
+    <section id="contact-section">
+        <div class="container alt-container">
+            <h2>
+                I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.
+            </h2>
+            
+            <img id="contact-bg" src="icon-img/contact-bg.svg" alt="">
+            
+            
+            
+            <form action="https://formspree.io/f/xknagbvo" method="post" id="contact-form">
+                <input type="text" name="name" id="client-name" maxlength="30" placeholder="Full Name" required>
+                <input type="email" name="email" id="client-email" placeholder="Email Address" required>
+                <textarea name="msg" id="client-msg" cols="30" rows="5" maxlength="500" placeholder="Leave a message" required></textarea>
+                <button class="action-btn" id="submit-btn" type="submit">Get in touch</button>
+            </form>
+            
+                
+            <ul>
+                <li><a class="media-link" href="https://twitter.com/abdallah_cu"><img src="icon-img/twitter.svg"
+                            alt="twitter logo"></a></li>
+                <li><a class="media-link"
+                        href="https://www.linkedin.com/in/abdallah-cuma-5a7617274/"><img
+                            src="icon-img/linkedin-icon.svg" alt="linked-in logo"></a></li>
+                <li><a class="media-link" href="https://medium.com"><img src="icon-img/medium.svg"
+                            alt="medium logo"></a></li>
+                <li><a class="media-link" href="https://github.com/AbdallahJumah95?tab=repositories"><img src="icon-img/github.svg"
+                            alt="github logo"></a></li>
+                <li><a class="media-link" href="https://angel.co/"><img src="icon-img/angellist.svg"
+                            alt="angellist logo"></a>
+                </li>
+            </ul>
+    
+            <span>abdualshaer6@gmail.com</span>
+    
+            <img id="contact-line-break" src="icon-img/line-shape.svg" alt="">
+        </div>
+
+        <div class="contact-bg-desk-container">
+            <img id="contact-bg-desk-1" src="icon-img/desktop/contact-bg-desktop-1.svg" alt="">
+            <img id="contact-bg-desk-2" src="icon-img/desktop/contact-bg-desktop-2.svg" alt="">
+            <img id="contact-bg-desk-3" src="icon-img/desktop/contact-bg-desktop-3.svg" alt="">
+        </div>
```

Comment:
- [ ] Great job adding images for the background of the desktop version, however, I noticed that they're also appearing on mobile devices. As this is a mobile project, it would be better to remove or hide these images for small screen sizes. Doing this will make your project look more professional and well-made. 💯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236656406-d7d9d208-4ea6-470c-9920-339b24a9fe9a.png)